tests/Entity/Post/Comment.php

```diff
@@ -25,6 +26,16 @@ public static function fields()
         ];
     }
 
+    public static function scopes()
+    {
+        return [
+            'yesterday' => function (Query $query) {
+                return $query->where(['date_created :gt' => date('Y-m-d H:i:s', strtotime('yesterday')), 'date_created :lt' => date('Y-m-d H:i:s', strtotime('today'))]);
```

Comment:
I made the change: https://github.com/vlucas/spot2/commit/4f0765345c0d91f3e27fdc7625f0c1536b964b76